autogen/agentchat/contrib/capabilities/generate_images.py

```diff
@@ -287,5 +299,9 @@ def _validate_resolution_format(resolution: str):
 
 
 def _validate_dalle_model(model: str):
-    if model not in ["dall-e-3", "dall-e-2"]:
-        raise ValueError(f"Invalid DALL-E model: {model}. Must be 'dall-e-3' or 'dall-e-2'")
+    if model not in VALID_DALLE_MODELS:
```

Comment:
For Azure models, they are not necessarily named as one of VALID_DALLE_MODELS. They can be named as anything.